kibble/models/item_index.go

```diff
@@ -83,6 +83,25 @@ func (itemIndex ItemIndex) Set(slug string, item GenericItem) {
 	}
 }
 
+// Set - an item
+func (itemIndex ItemIndex) SetWithStatus(slug string, statusID int, item GenericItem) {
```

Comment:
Just curious if there is a need to have both `Set` and `SetWithStatus`?

Reply:
There are a lot of uses of `itemIndex.Set` on items that do not really have an idea of Status ID yet, (although the generic item does) which I assume we would leave in until those items eventually have Status